source/win-assets/CustomCSS.vue

```diff
@@ -78,30 +89,55 @@ function saveCSS (): void {
   })
     .then(() => {
       lastLoadedCSS.value = editorContents.value
-      savingStatus.value = ''
+      setTimeout(() => { savingStatus.value = trans('Saved!') }, 1000)
+      setTimeout(() => { savingStatus.value = '' }, 2000)
     })
-    .catch(e => {
+    .catch(err => {
       savingStatus.value = trans('Saving failed')
-      console.error(e)
+      console.error(err)
     })
 }
 </script>
 
 <style lang="less">
-div#custom-css {
-  overflow: auto; // Enable scrolling, if necessary
-  padding: 10px;
-  width: 100vw;
+#css-container {
+  padding: 0px 10px;
   height: 100%;
   display: flex;
   flex-direction: column;
 
+  .admonition {
```

Comment:
I have thought long about it; I think it's nicer to wrap it in a div here instead of accessing a class name that may change at any time because it comes from a component.

(Short reasoning: I want to slowly move components to having no explicit margin, so that it can instead be provided by the containing element. The current status is a bit too opinionated and I didn't see that issue coming when I did my big UI rewrite a few years ago.)